src/components/SyncEditor/SyncEditorModal.tsx

```diff
@@ -116,14 +119,82 @@ const SyncEditorModal = ({
 
   return (
     <Modal
-      title="Edit YAML"
       isOpen={isOpen}
-      variant={ModalVariant.large}
+      variant="large"
```

Comment:
Nit: ModalVariant is available on the current Modal component so this line doesn't need to be changed.  If PF has a variable let's use it instead of a string.